LICENSE

```diff
@@ -0,0 +1,53 @@
+MIT License
+
+Copyright (c) 2021 Nathan Moore and [contributors](https://github.com/withastro/compiler/graphs/contributors)
```

Comment:
Some things are Fred and contributors, so instead of grabbing a second "Nate" special person we might want to rewrite this to be less person specific across the repo. Maybe:


```suggestion
Copyright (c) 2021 [Astro contributors](https://github.com/withastro/compiler/graphs/contributors)
```